validator/client/metrics.go

```diff
@@ -161,66 +161,109 @@ func (v *validator) LogValidatorGainsAndLosses(ctx context.Context, slot uint64)
 		}
 	}
 
-	included := 0
-	votedSource := 0
-	votedTarget := 0
-	votedHead := 0
 	prevEpoch := uint64(0)
 	if slot >= params.BeaconConfig().SlotsPerEpoch {
 		prevEpoch = (slot / params.BeaconConfig().SlotsPerEpoch) - 1
+		if v.voteStats.startEpoch == 0 {
+			//prevEpoch may be 0 so we store startEpoch (+1). Thus a value of 0 tells us it is not set (default)
```

Comment:
As Go's best practice, we should use 0 as real value. The undefined value can be treated us `^uint64(0)`.

Reply:
I'll fix it.  Thanks

Reply:
Addressed in 1347c82.  Thanks!